frontend/src/__tests__/cypress/cypress/tests/e2e/storageClasses/clusterStorage.cy.ts

```diff
@@ -12,15 +12,15 @@ const dspName = 'qe-cluster-storage-sc-dsp';
 
 describe('Regular Users can make use of the Storage Classes in the Cluster Storage tab from DSP ', () => {
   before(() => {
-    provisionClusterStorageSCFeature(dspName, TEST_USER.USERNAME);
+    provisionClusterStorageSCFeature(dspName, CONTRIBUTOR_USER.USERNAME);
```

Comment:
Same comment as previously.

Reply:
Fixed in [498164e](https://github.com/opendatahub-io/odh-dashboard/pull/3469/commits/498164e80896f458e9b029d87a341a52ddc1fddf)